### PR TITLE
DOC-5288: Fix incorrect logging config

### PR DESCRIPTION
**Pull request by JFlath**
Also enables console logging and uses default logging settings

### CODE REVIEW

File: modules/ROOT/assets/attachments/kubernetes/sgw-config-import.json [hunk 1]
```diff
@@ -1,32 +1,10 @@
...
-      },
-      "info": {
-        "enabled": false
-      },
-      "debug": {
-        "enabled": false
-      }
+      "enabled": true,
+      "log_level": "info",
+      "log_keys": ["*"]
```
Comment (bbrks) [inline modules/ROOT/assets/attachments/kubernetes/sgw-config-import.json:7]: Do we want to log `"*"` by default? What about just keeping the default of `"HTTP"`?

I wonder if keeping everything default/omitted, apart from `log_file_path` and `console.enabled` would simplify the config a bit?

**Review by bbrks — COMMENTED** [review]
What do you think about this?

File: modules/ROOT/assets/attachments/kubernetes/sgw-config-import.json (hunk 1 above)
Comment (JFlath) [reply modules/ROOT/assets/attachments/kubernetes/sgw-config-import.json:7]: I'm on the fence - I'm tempted to say `*` as it give a bit more feedback for those trying this for the first time (kinda who the linked page is aimed at). Having the params listed makes it nice and obvious what can be tuned down too...

Comment (bbrks) [reply modules/ROOT/assets/attachments/kubernetes/sgw-config-import.json:7]: That's fair enough - I guess with upcoming changes, even `"*"` at `INFO` shouldn't be too problematic.

**Review by jamesnocentini — APPROVED** [review]

**Pull request merged**